### PR TITLE
More touch support and VR performance fix

### PR DESCRIPTION
**Pull request by njbrown**

### CODE REVIEW

**Comment by iKlsR** [comment]
@njbrown can you please fix the author name commits........ and fix the .pro

**Comment by njbrown** [comment]
fixed all the issues @iKlsR

File: JahshakaVR.pro [hunk 1]
```diff
@@ -81,7 +81,11 @@ SOURCES += src/main.cpp\
     src/io/scenereader.cpp \
     src/widgets/propertywidgets/emitterpropertywidget.cpp \
     src/widgets/combobox.cpp \
+<<<<<<< HEAD
```
Comment (iKlsR) [inline JahshakaVR.pro]: Bad rebase

File: app/content/models/external_controller01_left.mtl [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+newmtl blinn1SG
+Ns 96.078431
+Ka 0.000000 0.000000 0.000000
+Kd 0.900000 0.900000 0.900000
+Ks 0.400000 0.400000 0.400000
+Ke 0.000000 0.000000 0.000000
+Ni 1.000000
+d 1.000000
+illum 2
+map_Kd C:\Users\Nicolas\Desktop\external_controllers\external_controller01_col.png
```
Comment (iKlsR) [inline app/content/models/external_controller01_left.mtl]: Woops? Absolute file

File: app/content/models/external_controller01_right.mtl [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+newmtl blinn1SG.001
+Ns 96.078431
+Ka 0.000000 0.000000 0.000000
+Kd 0.900000 0.900000 0.900000
+Ks 0.400000 0.400000 0.400000
+Ke 0.000000 0.000000 0.000000
+Ni 1.000000
+d 1.000000
+illum 2
+map_Kd C:\Users\Nicolas\Desktop\external_controllers\external_controller01_col.png
```
Comment (iKlsR) [inline app/content/models/external_controller01_right.mtl]: Absolute file

File: app/content/models/external_controller01_left.mtl (hunk 2 above)
Comment (njbrown) [inline app/content/models/external_controller01_left.mtl]: blender's obj exporter...
No need to worry about that, the texture is being set in code

**Comment by iKlsR** [comment]
Not sure why you squashed but ah well. There are three remaining issues. See my comments @njbrown

File: JahshakaVR.pro (hunk 1 above)
Comment (njbrown) [inline JahshakaVR.pro]: i can never seem to get this right...

**Comment by njbrown** [comment]
i squashed to make rebasing easier...and i still messed it up

**Comment by iKlsR** [comment]
LGTM 🤠

**Pull request merged**